File: BloodbankFrontend/src/app/modules/dto/passwordChangeDTO.ts
export class PasswordChangeDTO {
    newPassword!: string;
    confirmNewPassord!: string;
    userId!: number;


    public constructor(obj?: any) {
        if (obj) {
            this.newPassword = obj.newPassword;
            this.confirmNewPassord = obj.confirmNewPassord;
            this.userId = obj.userId;
        }
    }
}

File: BloodBank/src/main/java/com/isa/BloodBank/dto/PasswordChangeDTO.java
package com.isa.BloodBank.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class PasswordChangeDTO {
    private String newPassword;
    private String confirmNewPassord;
    private int userId;
}
